docs/reference.md

```diff
@@ -687,7 +687,7 @@ Posts a message to a stream. Probably the most commonly used activity to interac
 Key | Type | Required |
 ------------ | -------| --- | 
 [to](#to) | Map | No |
-[content](#send-message-content) | String | Yes |
+[content](#send-message-content) | String or Object | Yes |
```

Comment:
Then we should probably describe what the object is in #send-message-content

Reply:
I improved the doc in the last commit. What do you think?